.travis.yml

```diff
@@ -17,7 +16,7 @@ install:
   - '[[ -z "$CI_USER_TOKEN" ]] || composer config github-oauth.github.com ${CI_USER_TOKEN};'
   - travis_retry composer self-update && composer --version
   - travis_retry composer update --prefer-dist --no-interaction
-  - git clone -q --depth=1 -b 2.2 --recurse-submodules https://github.com/Naktibalda/codeception-zend-expressive-tests framework-tests
+  - git clone -q --depth=1 -b master --recurse-submodules https://github.com/Slamdunk/codeception-mezzio-tests framework-tests
```

Comment:
Do you want the ownership of the repo? I can transfer it to you any time

Reply:
no